### PR TITLE
[Uptime] synth package

### PR DESCRIPTION
**Pull request by justinkambic**
## What does this PR do?

We are doing some prototype work for synthetics to get a baseline package configured for our solution.

Our eventual goal is probably not to merge this exact code, but to aid us in the development of our central management story.

## How to test this PR locally

Follow the instructions under `testing/environments` to spin up the environment. The synthetics package should be included in the Ingest Manager when running this branch.

## Screenshots

image

### CODE REVIEW

File: packages/synthetics/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+config_templates:
+  - name: synthetics
+    title: Elastic Synthetics
+    description: Perform synthetic health checks on network endpoints.
+    inputs:
+      - type: synthetics/http
+        title: Monitor
+        description: Perform synthetic monitor checks
+# config_templates:
+#   - name: synthetics/http
```
Comment (ruflin) [inline packages/synthetics/manifest.yml:20]: Nice to see this prototyped on how it might work!

File: packages/synthetics/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+#     description: Perform uptime checks and monitor response times.
+
+#     # Kibana uses this field to link to a specific solution, and the field
+#     # can only be used by `solution` packages.
+#     solution: uptime
+#     multiple: false
+
+conditions:
+  kibana:
+    version: "^7.9.0"
```
Comment (ruflin) [inline packages/synthetics/manifest.yml:58]: Probably this should be ^7.10 at least?

File: packages/synthetics/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+config_templates:
+  - name: synthetics
+    title: Elastic Synthetics
+    description: Perform synthetic health checks on network endpoints.
+    inputs:
+      - type: synthetics/http
+        title: Monitor
+        description: Perform synthetic monitor checks
+# config_templates:
```
Comment (mtojek) [inline packages/synthetics/manifest.yml:19]: If this code is unused, please remove it from here.

**Comment by justinkambic** [comment]
@mtojek does https://github.com/elastic/integrations/pull/241/commits/2bb10ebccf1947be247d198032acb5271fe5c568 look ok for the owner entry? Our team has access to https://github.com/elastic/package-storage as well.

**Pull request merged**